internal/server/GameLoop.go

```diff
@@ -3,30 +3,115 @@ package server
 import (
 	"SOMAS2023/internal/common/objects"
 	"SOMAS2023/internal/common/physics"
+	"SOMAS2023/internal/common/utils"
 	"fmt"
+
+	"github.com/google/uuid"
 )
 
 func (s *Server) RunGameLoop() {
```

Comment:
This desperately needs to be split into functions and unit tested - it's very hard to read at the moment